test/e2e/media.spec.ts

```diff
@@ -61,4 +62,15 @@ describe('Media Metadata endpoints', () => {
     expect(res.data).toHaveProperty('trivia');
     expect(res.data).toHaveProperty('tagline');
   });
+
+  it('should create a failed lookup document when Open Subtitles cannot find metadata', async() => {
+    await FailedLookupsModel.deleteMany({});
+    try {
+      await axios(`${appUrl}/api/media/f4245d9379d31e30/1234`);
+    } catch (e) {
+      const doc = await FailedLookupsModel.findOne({ osdbHash: 'f4245d9379d31e300' });
```

Comment:
@js-kyle this test isn't doing its job - its assertions are inside the `catch` block but that isn't being entered, because the API doesn't return an error in that case - it returns the status 200 and 0 results.
The test passes because it is just doing line 69 and ignoring the rest

Reply:
@js-kyle I'm fixing it